src/components/ModalWindow.vue

```diff
@@ -4,70 +4,76 @@
       id="modal-scrollable"
       size="lg"
       scrollable
-      title="Деталі фільму"
+      title="Movie Details"
       :hide-footer="true"
       body-class="modal-window"
     >
-      <div class="movie-details d-flex">
-        <div class="d-flex justify-content-center">
-          <img
-            width="300"
-            class="movie-details__poster"
-            :src="
-              movieDetails.poster_path === null
-                ? 'https://st4.depositphotos.com/17828278/24401/v/600/depositphotos_244011872-stock-illustration-image-vector-symbol-missing-available.jpg'
-                : 'https://image.tmdb.org/t/p/original' +
-                  movieDetails.poster_path
-            "
-            alt="poster"
-          />
-        </div>
-        <div class="wrapper-details">
-          <div class="details-info">
-            <h2>{{ movieDetails.title }}</h2>
-            <div class="details-item">
-              <div class="details-subtitle">Жанри:</div>
-              <div class="details-subitem">{{ genres }}</div>
-            </div>
-            <div class="details-item">
-              <div class="details-subtitle">Дата виходу:</div>
-              <div class="details-subitem">{{ movieDetails.release_date }}</div>
-            </div>
-            <div class="details-item">
-              <div class="details-subtitle">Рейтинг:</div>
-              <div class="details-subitem">
-                {{ movieDetails.vote_average }} / {{ movieDetails.vote_count }}
+      <div class="movie-details d-flex flex-column">
+        <div class="d-flex flex-row">
+          <div class="d-flex justify-content-center">
+            <img
+              width="300"
+              class="movie-details__poster"
+              :src="
+                !movieDetails.poster_path
+                  ? 'https://st4.depositphotos.com/17828278/24401/v/600/depositphotos_244011872-stock-illustration-image-vector-symbol-missing-available.jpg'
```

Comment:
Move these URLs into constant `data` variables.

Use CAPS_CASE for variable names so you know they aren't dynamic.

Add a code comment explaining what the URL is for.